package.json

```diff
@@ -25,5 +28,8 @@
   "engine": {
     "node": ">=0.8.17"
   },
-  "author": "omphalos"
+  "author": "omphalos",
+  "dependencies": {
+    "nodeunit": "^0.9.1"
+  }
```

Comment:
This definitely should be part of the package.json.  Would you mind making this a dev dependency though?